.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Background
+<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+
+## Definition of Done
+
+- [ ] _Replace this line with individual tasks unique to your PR_
+- [ ] Appropriate test coverage & logging
```

Comment:
I think we have test coverage enforced here, do we need to mention that?

Reply:
@jbalboni brings up a good point.  What does every think:  keep the reminder task, or allow the Jenkins test coverage failure prompt to serve as the reminder?

Reply:
I can see some value in leaving, but am open to either. I will sometimes create a PR before I'm completely done with tests so that my team can give me early feedback on the approach -- I could see a use case for this flag in a situation like that.